threatstream_connector.py

```diff
@@ -652,6 +654,7 @@ def _domain_reputation(self, param):
     def _ip_reputation(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
         value = param[THREATSTREAM_JSON_IP]
+        extend_source = param.get("extend_source", False)
```

Comment:
I believe the extend_source is not being used (see https://github.com/splunk-soar-connectors/threatstream/blob/f4639f71065827c21d329dab883d25b494018b01/threatstream_connector.py#L657)